tests/framework/widgets/DetailViewTest.php

```
@@ -311,8 +308,8 @@ public function testOptionsTags(): void
             ],
         ]);
 
-        foreach ($this->detailView->attributes as $index => $attribute) {
-            $a = $this->detailView->renderAttribute($attribute, $index);
+        foreach ($detailView->attributes as $index => $attribute) {
+            $a = $detailView->renderAttribute($attribute, $index);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $a. Configured minimum length is 3. (undefined)

(ShortVariable)